azurerm/import_arm_action_group_test.go

```diff
@@ -0,0 +1,77 @@
+package azurerm
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMActionGroup_importBasic(t *testing.T) {
```

Comment:
Minor, you could move the import tests in with the regular tests if you would like. It looks like this is the direction that other providers are going, so we may want to do this as well.

Reply:
Moved to `resource_arm_action_group_test.go`